src/tips/topics/configue-togg-auto.md

```diff
@@ -0,0 +1,19 @@
+# Updated menu Site Toggling and Automation
+
+Click **Dark Reader** icon.
+
+<img src="/images/tips/reset-settings-icon.png" alt="Dark Reader icon" style="width: 16rem;" loading="lazy" />
+
+On the top menu you will see two updated settings:
+
+<img src="/images/tips/configue-main.png" style="width: 16rem;" loading="lazy" />
+
+Press on the left settings and customize Site Toggling.
+
+<img src="/images/tips/configue-site-toggling.png" style="width: 16rem;" loading="lazy" />
+
+Enabled for all websites by default and detect the website’s dark theme, also customize your keyboard shortcut.
```

Comment:
- If you want Dark Reader to work only on certain websites, uncheck the *Enabled by default** checkbox.
- Some websites can already have dark theme implemented. Use **Detect dark theme** option to stop Dark Reader from interfering with the website's built-in dark theme.
- Customize a keyboard shortcut for quicker website toggling.